src/transformers/tokenization_utils_base.py

```diff
@@ -4072,16 +4072,16 @@ def clean_up_tokenization(out_string: str) -> str:
             `str`: The cleaned-up string.
         """
         out_string = (
-            out_string.replace(" .", ".")
+            out_string.replace(" ' ", "'")
```

Comment:
This function is deprecated anyways, and changing this will break backward compatibility!

Reply:
However it sounds like a decent idea 😉

Reply:
thanks